src/ImageSearchModal.jsx

```diff
@@ -15,238 +15,203 @@ import cockpit from 'cockpit';
+        Dialogs.close();
+    };
+
+    return (
+        <Modal isOpen className="podman-search"
+               position="top" variant="large"
+               onClose={handleClose}
+               title={_("Search for an image")}
+               footer={<>
+                   <Form isHorizontal className="image-search-tag-form">
+                       <FormGroup fieldId="image-search-tag" label={_("Tag")}>
+                           <TextInput className="image-tag-entry"
+                                  id="image-search-tag"
+                                  type='text'
+                                  placeholder="latest"
+                                  value={imageTag || ''}
+                                  onChange={(_event, value) => setImageTag(value)} />
+                       </FormGroup>
+                   </Form>
+                   <Button variant='primary' isDisabled={selected === ""} onClick={onDownloadClicked}>
+                       {_("Download")}
+                   </Button>
+                   <Button variant='link' className='btn-cancel' onClick={handleClose}>
+                       {_("Cancel")}
+                   </Button>
+               </>}
+        >
+            <Form isHorizontal>
+                {dialogError && <ErrorNotification errorMessage={dialogError} errorDetail={dialogErrorDetail} />}
+                { userServiceAvailable && systemServiceAvailable &&
+                <FormGroup id="as-user" label={_("Owner")} isInline>
+                    <Radio name="user" value="system" id="system" onChange={onToggleUser} isChecked={isSystem} label={_("system")} />
+                    <Radio name="user" value="user" id="user" onChange={onToggleUser} isChecked={!isSystem} label={user} />
+                </FormGroup>}
+                <Flex spaceItems={{ default: 'inlineFlex', modifier: 'spaceItemsXl' }}>
+                    <FormGroup fieldId="search-image-dialog-name" label={_("Search for")}>
+                        <TextInput id='search-image-dialog-name'
+                                   type='text'
+                                   placeholder={_("Search by name or description")}
+                                   value={imageIdentifier}
+                                   onKeyDown={onKeyPress}
+                                   onChange={(_event, value) => setImageIdentifier(value)} />
+                    </FormGroup>
+                    <FormGroup fieldId="registry-select" label={_("in")}>
+                        <FormSelect id='registry-select'
+                            value={selectedRegistry}
+                            onChange={(_ev, value) => { setSelectedRegistry(value); clearTimeout(typingTimeout); onSearchTriggered(value, false) }}>
+                            <FormSelectOption value="" key="all" label={_("All registries")} />
+                            {(searchRegistries || []).map(r => <FormSelectOption value={r} key={r} label={r} />)}
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1332-20230705-092041-36f39867-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)